arcgis-ios-sdk-samples/AppDelegate.swift

```diff
@@ -180,3 +183,25 @@ extension UIColor {
     // The translucent background color for status labels.
     class var statusLabelBackgroundColor: UIColor { return UIColor(named: "statusLabelBackgroundColor")! }
 }
+
+extension AppDelegate {
+    /// License the app with ArcGIS Runtime deployment license keys.
+    ///
+    /// - Note: An invalid key does not throw an exception, but simply fails to license the app,
+    ///   falling back to Developer Mode (which will display a watermark on the map view).
+    static func licenseApplication() {
+        do {
+            // Don't set extension key if you don't use utility network samples.
+            // try AGSArcGISRuntimeEnvironment.setLicenseKey(.licenseKey)
+            
+            // Set both keys for accessing all samples.
+            try AGSArcGISRuntimeEnvironment.setLicenseKey(.licenseKey, extensions: [.extensionLicenseKey])
+            
+            // Set API keys for new metered basemap styles.
+            // AGSArcGISRuntimeEnvironment.apiKey = .apiKey
+        } catch {
+            print("[Error: AGSArcGISRuntimeEnvironment] Error licensing app: \(error.localizedDescription)")
+        }
+        print("[ArcGIS Runtime License] \(AGSArcGISRuntimeEnvironment.license())")
```

Comment:
The prints are not ideal - I copied from Data Collection app. We should discuss how to handle licensing success/failture.

Reply:
Please note, there are a few swift extensions at play that make printing the license user friendly. I strongly suggest you include [these extensions](https://github.com/Esri/data-collection-ios/blob/eli/clean-extensions/data-collection/data-collection/AppDelegate.swift#L88-L201). to really get the most out of `licenseApplication()`.

Please note, in Data Collection `master` branch, they are buried in an extensions group. However, they are being moved to be more noticeable. This is a WIP.